app.go

```diff
@@ -18,7 +19,7 @@ func run() {
 
 	httpServer = gin.Default()
 
-	routes.RegisterRoutes(httpServer)
+	router.RegisterRouter(httpServer)
```

Comment:
hmm, I think that the method "RegisterRouter" just be named as "Register" is better, because it's defined under the router.

Reply:
I've fixed it.